### ISSUE TITLE
Get rid of "list of all configuration variables"

### ISSUE THREAD

**Issue by rdb** [panda3d/panda3d-docs#27]
A reference of all variables has no place in a manual.  Instead, such a list should be automatically generated as part of the Reference (see #21).

We could have a list of variables under each module generated by the API reference generator.  Then, we can create cross-references to the documentation for each particular variable from the manual text where this variable is mentioned.

**Comment by Moguri** [comment]
I am fine with not having the page in the manual proper, but I *do* reference this page a lot, so some form of replacement would be appreciated.

**Comment by rdb** [comment]
Yes, I think it should be automatically generated in the reference section.  I don't intend to remove it before we have a replacement.
